buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/LicenseHeadersTask.groovy

```diff
@@ -33,6 +36,13 @@ import java.nio.file.Files
  */
 public class LicenseHeadersTask extends AntTask {
 
+    // checks all the java sources (allJava)
+    @InputFiles
+    Iterable<File> input = project.sourceSets.collect({it.allJava.srcDirs}).flatten()
+
+    @OutputFile
+    File reportFile = new File(project.buildDir, 'reports/licenseHeaders/markers/jarHell')
```

Comment:
This seems like a very odd filename...why can't it stay as reports/licenseHeaders/rat.log?